copyright

```diff
@@ -1490,6 +1498,13 @@ Copyright: Becca Tommaso
 License: CC0 (Creative Commons Zero)/Public-Domain
 Comment: Derived from works by ESA/Rosetta spacecraft (under the same license).
 
+Files:
+ images/outfit/anti-materiel?gun*
```

Comment:
The AMG isn't really based on any pre-existing model made by MZ(or anyone else) so I don't think it's considered a derivative work and might have to be on its own block. Might be the same for Decoy Plating and Refuelling Module by Becca which I don't think resemble anything before, not too sure though and might have to just ask him.

Reply:
I know the Refuelling Module is derived because it's got that shape of the top of the Reactor Modules.
Will ask for the Decoy Plating though, and move the AMG.